applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/rules.js

```diff
@@ -349,6 +349,10 @@ return view.extend({
 		o.allowany = true;
 		o.allowlocal = 'src';
 
+		o = s.taboption('general', form.Value, 'ipset', _('Use ipset'));
```

Comment:
We should loop uci sections of type `ipset` and present their names here, like

```javascript
o = s.taboption('advanced', form.Value, 'ipset', _('Use ipset'));
uci.sections('firewall', 'ipset', function(s) {
    if (typeof(s.name) == 'string')
            this.value(s.name, s.comment ? '%s (%s)'.format(s.name, s.comment) : s.name);
});
```

Reply:
Nice addition. Borrowed.